react-example/src/components/Arkose.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Arkose extends React.Component {
+  constructor () {
+    super();
+    this.myEnforcement = null;
+  }
+
+  // Append the JS tag to the Document Body.
+  loadScript = () => {
+    const scriptId = 'arkose-script';
```

Comment:
probably an idea to append the public key to the script id, because if theres multiple ECs with different keys on the same page this could remove the incorrect key's script